Find a functionally equivalent code:com.android.connectivitymanagertest.ConnectivityManagerTestBase.turnScreenOn:COMMENT
<com.android.connectivitymanagertest.ConnectivityManagerTestBase: void turnScreenOn()>
Method Modifier: public      protected   
Method Modifier: Yes         No          
Method Updated:  No          Yes         
Comment:// Turn screen on

@@ -1,5 +1,5 @@
 {
-    log("Turn screen on");
+    logv("Turn screen on");
     PowerManager pm = (PowerManager) mContext.getSystemService(Context.POWER_SERVICE);
     pm.wakeUp(SystemClock.uptimeMillis());
     // disable lock screen

